src/core/manager.c

```diff
@@ -949,10 +949,14 @@ int manager_new(LookupScope scope, ManagerTestRunFlags test_run_flags, Manager *
                         return r;
         }
 
-        m->taint_usr =
+        m->taint_split_usr =
                 !in_initrd() &&
                 dir_is_empty("/usr") > 0;
 
+        m->taint_unmerged_usr =
```

Comment:
no need to cache this. we can check this cheaply any time...

The reason why the split usr thing is cached was that it reports whether /usr is initially empty when pid 1 initializes, and thus it must be done early since if we do it later we can't really do the test anymore since something most likely will be mounted over /usr/.

Hence, just do the check from manager_taint_string()